docs/tips/03-if-else-in-JSX.md

```diff
@@ -30,7 +30,7 @@ React.createElement("div", {id: if (condition) { 'msg' }}, "Hello World!");
 That's not valid JS. You probably want to make use of a ternary expression:
 
 ```js
-ReactDOM.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
+ReactDOM.render(<div id={condition ? 'msg' : null}>Hello World!</div>, mountNode);
```

Comment:
Nah, we have no intention of null and empty string being the same. They are very different values. In this case the empty string will create markup with `<div id="">` whereas null will be `<div>`. While it might not matter for ids, it's not a safe assumption that all attributes will treat those 2 cases the same.

Anyway, I'll take this since the change is good. Thanks!